VAMobile/src/store/slices/claimsAndAppealsSlice.ts

```diff
@@ -112,9 +112,11 @@ const emptyClaimsAndAppealsGetData: api.ClaimsAndAppealsGetData = {
 }
 
 export const sortByLatestDate = (claimsAndAppeals: ClaimsAndAppealsList): ClaimsAndAppealsList => {
-  return sortBy(claimsAndAppeals || [], (claimAndAppeal) => {
-    return new Date(claimAndAppeal.attributes.updatedAt)
-  }).reverse()
+  return chain(claimsAndAppeals)
```

Comment:
Cool, haven't seen this sorting method before

Reply:
I picked this one up when I did that big `lodash/fp` removal in vets-website haha. Underscore works exactly the same, which is nice